extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -91,7 +99,7 @@ const Track = memo(
 					className="jetpack-podcast-player__track-link"
 					href={ track.link }
 					role="button"
-					aria-pressed="false"
+					aria-current={ isActive ? __( 'track' ) : undefined }
```

Comment:
Is that supposed to get translated? If it does it would need the `jetpack` text domain and probably a context for translators.

Reply:
It can get announced in some screen readers. It's supposed to announce something like "[button text], current [aria-current value]". So, in this case it would be "[button text] current track".  Some screen readers just announce "current item" instead of using the `aria-current` value though.

I think it needs to be a one word thing. I'll update it to use `__x()` with context.

Reply:
https://github.com/Automattic/jetpack/pull/15207#discussion_r401442817